app/utils/errors.py

```diff
@@ -11,7 +11,17 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-"""Functions to handle errors data structure."""
+"""Functions and classes to handle errors data structure."""
+
+
+class BackendError(Exception):
+
+    def __init__(self, errors):
+        self.errors = errors
+
+    def __str__(self):
+        return ', '.join([' '.join([str(k)] + v)
```

Comment:
Minor nitpick: can you please use double quotes? We basically use only double quotes in the code base.